sechub-sereco/src/test/java/com/daimler/sechub/sereco/test/AssertVulnerabilities.java

```diff
@@ -284,6 +345,49 @@ public VulnerabilityFinder and() {
 
     }
 
+    private static void dumpOnSystemOutCWEBased(List<SerecoVulnerability> vulnerabilities) {
```

Comment:
What is the purpose of this method? Why does this method writes to `System.out`?

Reply:
The purpose for this is, too dump given vulnerabilities CWE based to system out when our test condition fails.

When you look at the call hierarchy you see, that our isContained and isNotContained methods of the test assert object will print this out as additional metainformation for the developer to understand the data structure and fix the problems. 

I did not add it to an exception message because the data is most time too big. Because every unit test does also includes its system out this is absolutely okay.

Reply:
Thanks for the explanation.